theano/__init__.py

```diff
@@ -67,7 +67,7 @@
     predefined_modes, predefined_linkers, predefined_optimizers,
     FunctionMaker, function, function_dump, OpFromGraph,
     ProfileMode, ProfileStats,
-    Param, shared, as_op)
```

Comment:
What about making paran a sub class or in and adding the missing, changed
name parameter to init?

Le mar. 26 janv. 2016 14:34, Pascal Lamblin <notifications@github.com> a
écrit :

> In theano/__init__.py
> <https://github.com/Theano/Theano/pull/3890#discussion_r50912251>:
>
> > @@ -67,7 +67,7 @@
> >      predefined_modes, predefined_linkers, predefined_optimizers,
> >      FunctionMaker, function, function_dump, OpFromGraph,
> >      ProfileMode, ProfileStats,
> > -    Param, shared, as_op)
>
> Actually, this is breaking something in the deep learning tutorials.
> Should we keep an alias Param = In, or a dummy Param class that calls In
> with a deprecation warning?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Theano/Theano/pull/3890/files#r50912251>.
>